pygrocy/grocy.py

```diff
@@ -11,12 +11,21 @@
 
 
 class Product(object):
-    def __init__(self, stock_response: CurrentStockResponse):
-        self._product_id = stock_response.product_id
-        self._available_amount = stock_response.amount
-        self._best_before_date = stock_response.best_before_date
-
-        self._name = None
+    def __init__(self, response):
+        if type(response) is CurrentStockResponse:
+            self._product_id = response.product_id
+            self._name = None
+            self._available_amount = response.amount
+            self._best_before_date = response.best_before_date
+            self._amount_missing = None
+            self._is_partly_in_stock = None
+        elif type(response) is MissingProductResponse:
+            self._product_id = response.product_id
+            self._name = response.name
+            self._available_amount = None
+            self._best_before_date = None
+            self._amount_missing = response.amount_missing
+            self._is_partly_in_stock = response.is_partly_in_stock
```

Comment:
`is`suggests that this is respresenting a boolean. Could you investigate that a abit?

Reply:
I thought the same, but Grocy documentation is set as an Integer:

```
missing_products : [ 
   id | integer
   name | string
   amount_missing | number
   is_partly_in_stock | integer ]
```

Reply:
In [0038.sql](https://github.com/grocy/grocy/blob/b81316bd60b3ff96a0bd5add73bbf0a1fbf8dece/migrations/0038.sql) it seems to be a 0 - 1 integer so I think it's safe to change it to boolean. I'll update the PR.